internal/scanner/scanner.go

```diff
@@ -109,6 +113,67 @@ func (s *Scanner) ScanNamespace(nsName string) error {
 	return s.reportStore.SaveAll()
 }
 
+func (s *Scanner) ScanClusterWideResources() error {
+	log.Info().Msg("cluster wide scan started")
+	policies, skippedNum, err := s.policiesFetcher.GetClusterAdmissionPolicies()
+	if err != nil {
+		return err
+	}
+	log.Debug().
+		Dict("dict", zerolog.Dict().
+			Int("policies to evaluate", len(policies)).
+			Int("policies skipped", skippedNum),
+		).Msg("cluster admission policies count")
+	auditableResources, err := s.resourcesFetcher.GetClusterWideResourcesForPolicies(context.Background(), policies)
+	if err != nil {
+		return err
+	}
+	// create PolicyReport
+	clusterReport := report.NewClusterPolicyReport("cluster-report")
+	clusterReport.Summary.Skip = skippedNum
+	// Iterate through all auditableResources. Each item contains a list of resources and the policies that would need
+	// to evaluate them.
+	for _, resource := range auditableResources {
+		for _, policy := range resource.Policies {
```

Comment:
Optional nitpicking, feel free to ignore: I would create a new function for this loop, analogous to `auditResource()`, (e.g: clusterAuditResource() or `auditClusterResource()`). This would reduce the complexity of `ScanClusterWideResources()`, and separate the "map" and "reduce" logic.

Reply:
Yep, I started the same interface on a previous PR, but we removed it at the end. I'm ok adding it later on.